bower.json

```diff
@@ -35,7 +35,7 @@
   "dependencies": {
     "polymer": "^2.0.1",
     "myscript-common-element": "4.1.x",
-    "KaTeX": "katex#^0.8.3"
+    "KaTeX": "^0.8.3"
```

Comment:
This should be `"katex": "^0.8.3"` since `KaTeX` isn't a valid Bower artifact name.